src/process.cr

```diff
@@ -258,26 +158,8 @@ class Process
 
     reader_pipe, writer_pipe = IO.pipe
```

Comment:
I think the `reader_pipe`/`writer_pipe` stuff should be moved into `Crystal::System`, it's platform-specific. WIndows has no need for it: there's no fork.

Reply:
Yes i can do it.